### PR TITLE
Full screen position issue

### PR DESCRIPTION
**Pull request by Arjun-Dagar**

### CODE REVIEW

**Comment by therealansh** [comment]
Hi @Arjun-Dagar , the logo in the body(circle one) seems to misbehaves in diff sizes. The idea is to intersect the green arc with the grey circle(a lil bit).

File: src/styles/Home.css [hunk 1]
```diff
@@ -7,7 +7,8 @@
 	text-align: center;
 	align-items: center;
 	left: 10%;
-	position: fixed;
+	position: absolute;
+	top: 20%;
```
Comment (therealansh) [inline src/styles/Home.css:12]: This is not required acc to the design the text needs to be in center.

File: src/pages/Home.js [hunk 2]
```diff
@@ -29,7 +29,7 @@ export default function Home() {
         style={{ backgroundColor: "#c8ffe4", borderRadius: "100%" }}
       >
         <div className="logo-dsc" style={{ backgroundColor: "#f0f0f0" }}>
-          <img src={logo} style={{ width: "100%" }}></img>
+          <img src={logo} style={{ width: "40%" }}></img>
```
Comment (therealansh) [inline src/pages/Home.js:33]: this will mess up the alignment of the image in respect to the grey circle.

**Review by therealansh — CHANGES_REQUESTED** [review]

**Comment by Arjun-Dagar** [comment]
Sorry, But i am not able to understand the issue with logo in body area.

**Pull request closed without merging**